### PR TITLE
Adding new license type for az sql vm

### PR DESCRIPTION
**Pull request by yareyes**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yareyes** [comment]
@Juliehzl could you please help merge this PR? Thanks

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -229,6 +229,10 @@ Release History
 * Remove unrelated exception from error message
 * Fix issues with incorrect error message "You do not have the required permissions needed to perform this operation." when blocked by network rules or AuthenticationFailed.
 
+**SQL VM**
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: Please rebase dev branch and place your test above 2.0.78 for next release.

**Comment by Juliehzl** [comment]
LGTM in general. Please resolve my comment and if CI agrees I will approve.

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (yareyes) [reply src/azure-cli/HISTORY.rst]: Done. Thank you

**Comment by yareyes** [comment]
@Juliehzl comment resolved. Please merge when convenient :)

**Pull request merged**